internal/query/projection/user.go

```diff
@@ -135,13 +137,15 @@ func (*userProjection) Init() *old_handler.Check {
 			handler.NewColumn(NotifyInstanceIDCol, handler.ColumnTypeText),
 			handler.NewColumn(NotifyLastEmailCol, handler.ColumnTypeText, handler.Nullable()),
 			handler.NewColumn(NotifyVerifiedEmailCol, handler.ColumnTypeText, handler.Nullable()),
+			handler.NewColumn(NotifyVerifiedEmailLowerCol, handler.ColumnTypeText, handler.Nullable()),
```

Comment:
i'd prefer a computed (stored) column to prevent missing changes

Reply:
I'll look into if it's possible with our implementation to create stored columns.

Reply:
In the setup step the value is computed yes, but the column from where it gets computed is filled as a copy form another col, the current implementation just adds the copy to the NotifyVerifiedEmailLowerCol.
So the result is exactly the same, either it gets copied as well or it is empty, as I don't think we can create stored columns for now.